docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,177 @@
+| `Copyright`                 | `Copyright`                  | `copyright`                 | Copyright details for the package. language.                                                                      |
+| `Licensing - Expression`    | `PackageLicenseExpression`   | `license type="expression"` | An SPDX license expression or path to a license file within the package, often shown in UIs like NuGet.org.       |
+| `Licensing - File`          | `PackageLicenseFile`         | `license type="file"`       | File for custom license that isn't supported by license expressions                                               |
+| `Project URL`               | `PackageProjectUrl`          | `projectUrl`                | A URL for the project homepage.                                                                                   |
+| `Icon File`                 | `PackageIcon`                | `icon`                      | An image to use as the icon for the package.                                                                      |
+| `Repository URL`            | `RepositoryUrl`              | `repository url=""`         | URL to the repository from which the package was built.                                                           |
+| `Repository type`           | `RespositoryType`            | `repository type=""`        | Type of repository the repository URL is pointing to (i.e. git)                                                   |
+| `Tags`                      | `PackageTags`                | `tags`                      | A space-delimited list of tags and keywords that describe the package. Tags are used when searching for packages. |
+| `Release notes`             | `PackageReleaseNotes`        | `releaseNotes`              | A description of the changes made in this release of the package.                                                 |
+
+### Package ID
+
+If you're publishing a completely new package:
+
+✔️ DO choose a package ID that is unique and clearly differentiated from existing packages on NuGet.org.
+> You can make sure a package ID is unique and differntiable by searching for the ID on NuGet.org or checking if the following link exists: https://www.nuget.org/packages/<package name\>.
+
+✔️ CONSIDER choosing a NuGet package name with a prefix that meets NuGet's prefix reservation [criteria](https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation#id-prefix-reservation-criteria).
+> Reserving the prefix ID for your package will let you get the verified check mark:
+> ![image](media/Verified-check-mark.png)
+> Check out the [Package ID prefix reservation docs](https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation) to learn more.
+
+### Package Version
+
+✔️ CONSIDER using [SemVer 2.0.0](https://semver.org/) to version your NuGet package.
+> Essentially, this means using the Major.Minor.Patch[-prerelease] format.
+
+✔️ DO include a pre-release suffix when releasing a non-stable package.
```

Comment:
https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/nuget#pre-release-packages also has some information we could use?